voluncheer/voluncheer/settings.py

```diff
@@ -172,10 +177,11 @@
 EMAIL_BACKEND = "django_ses.SESBackend"
 AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY")
 AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
-AWS_SES_REGION_NAME = "us-west-2"  # (ex: us-east-2)
+AWS_SES_REGION_NAME = "us-east-2"  # (ex: us-east-2)
```

Comment:
These comments are exactly the same as the actual content and thus don't really add anything.

```suggestion
AWS_SES_REGION_NAME = "us-east-2"
AWS_SES_REGION_ENDPOINT = "email.us-east-2.amazonawz.com"
```

Reply:
Noted. This is temporary since Hanna are going to change the email verification mechanism.